### PR TITLE
chore: (deps-dev) update commitlint rules

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a brief summary of this pull request.
Updated commitlint rules: 
- body-max-line-length: 200
- footer-max-line-length: 200

#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [X] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [X] Run npm run build-pack-library and test in external application

Documentation checklist:
- [X] update `README.md`
- [X] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

**Review by droshev — APPROVED** [review]

File: commitlint.config.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ const Configuration = {
             'always',
             ['build', 'ci', 'docs', 'feat', 'fix', 'style', 'refactor', 'test', 'revert', 'chore']
         ],
-        'scope-enum': [2, 'always', ['Core', 'Platform', 'doc-app', 'release', 'e2e', 'deps', 'deps-dev']]
+        'scope-enum': [2, 'always', ['Core', 'Platform', 'doc-app', 'release', 'e2e', 'deps', 'deps-dev']],
```
Comment (platon-rov) [inline commitlint.config.js:10]: Will it work for `(Core|Platform)` cases?

Comment (dimamarksman) [reply commitlint.config.js:10]: Why do we have `Core`, `Platform` starting from uppercase and others not? 
I supposed that the scope should always start from uppercase.

Comment (Betrozov) [reply commitlint.config.js:10]: > Will it work for `(Core|Platform)` cases?

yes

Comment (Betrozov) [reply commitlint.config.js:10]: >  Why do we have `Core`, `Platform` starting from uppercase and others not?
> I supposed that the scope should always start from uppercase.

part of these scopes generated through bots and I can not influence this
that is why we have part of these scopes lowercase

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**